docs/user-guide/table-management.md

```diff
@@ -45,6 +92,26 @@ to tags in other time-series systems like [InfluxDB][1].
 
 > TBD:  Creating table in distributed mode.
```

Comment:
Please also remove this line. There's no difference bwteen standalone and distributed mode regarding creating tables.